tests/utils.js

```diff
@@ -139,7 +139,7 @@ const deleteAll = (except) => {
     'branding',
     'partners',
     'settings',
-    /^form:contact:/,
+    /^form:/,
```

Comment:
Yeap, it was a little hard for me to find it 😅

Reply:
So, this is random, @tatilepizs, but I just want to make sure you know your debugging work here was not in vain!

I was just seeing weird issues with my new tests working when I ran them on their own, but failing when I ran the whole suite.  Eventually I realized it was because the "New Action" button was disabled.  I was very confused by what was going on, but after awhile I remembered that this sounded very similar to the issue you had seen.  Looked up this code change again and dropped it in my branch.  Sure enough!  It seems to have solved my issue!  (Without me having to debug for hours to find a fix :sweat: )  :partying_face:

Reply:
Thank you for sharing it! 😄 It makes me really happy that this change was able to help you! 🎉